### PR TITLE
Added video listing and single video page.

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Created videolisting page and single video page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,13 +2,17 @@ import "./App.css";
 import { Signin } from "./Pages/Authentication/Login/login";
 import { Routes , Route } from "react-router-dom"
 import { Signup } from "./Pages/Authentication/Signup/signup";
+import { VideoListing } from "./Pages/Videolisting/videolisting";
+import { SingleVideoPage } from "./Pages/SigleVideoPage/single-video";
```
Comment (sharmayatendra) [inline src/App.js]: you can use single import export file

File: src/Pages/SigleVideoPage/single-video.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+.avatar{
+    max-width: 1.7rem;
+    min-height: 1.7rem;
+    border-radius: 50%;
+    margin-left: 0.5rem;
+}
+
+.modal-card{
+    border: 1px solid rgb(212, 54, 54);
```
Comment (sharmayatendra) [inline src/Pages/SigleVideoPage/single-video.css:56]: you can use color variable here as well

File: src/Pages/SigleVideoPage/single-video.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    width: fit-content;
+    height: 10rem;
+    position: relative;
+    left: 10rem;
+}
+
+.modal-card>input{
+    margin : 1rem 1rem 0rem 1rem ;
+    width: fit-content;
+    border-bottom: 1px solid black;
```
Comment (sharmayatendra) [inline src/Pages/SigleVideoPage/single-video.css:66]: same here no need to hardcode the colors

File: src/Pages/Videolisting/videolisting.css [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+.delete-icon{
+    cursor: pointer;
+    position: relative;
+    bottom: 1.6rem;
+    left: 14rem;
+}
+
+.clear-history-btn{
+    font-size: 1.2rem;
+    color : rgb(212, 54, 54);
```
Comment (sharmayatendra) [inline src/Pages/Videolisting/videolisting.css]: same as above

File: src/Pages/Videolisting/videolisting.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
+import { Header } from "../../Components/header"
+import axios from "axios"
+import { useEffect } from "react"
+// import "./videolisting.css"
```
Comment (sharmayatendra) [inline src/Pages/Videolisting/videolisting.js]: remove commented code

**Review by sharmayatendra — APPROVED** [review]
you can work upon suggested changes good work

**Review by chanchal-panpaliya — APPROVED** [review]
1. code looks clean with proper naming 
2. good UI

**Pull request merged**